### PR TITLE
Block CDC datasource for NC counties

### PR DESCRIPTION
**Pull request by smcclure17**
Block CDC data so that state-dashboard data can flow through for North Carolina counties.

### CODE REVIEW

File: libs/datasets/combined_datasets.py [hunk 1]
```diff
@@ -158,6 +158,11 @@ def datasource_regions(
 # so this region mask might not always be in use.
 NE_COUNTIES = RegionMask(AggregationLevel.COUNTY, states=["NE"])
 
+
+# 1/27/2020: North Carolina CDC data is inaccurate due to issues tracking
```
Comment (mikelehen) [inline libs/datasets/combined_datasets.py:162]: 2020? 😛

File: libs/datasets/combined_datasets.py [hunk 2]
```diff
@@ -216,15 +221,16 @@ def datasource_regions(
     ]
 )
 CDCNewVaccinesCompletedBoosterCountiesWithoutExceptions = datasource_regions(
-    CDCNewVaccinesCountiesDataset, exclude=[CDC_STATE_EXCLUSIONS, *CDC_COUNTY_EXCLUSIONS]
+    CDCNewVaccinesCountiesDataset,
+    exclude=[CDC_STATE_EXCLUSIONS, *CDC_COUNTY_EXCLUSIONS, NC_COUNTIES],
```
Comment (mikelehen) [inline libs/datasets/combined_datasets.py:225]: Is there a reason not to just add NC_COUNTIES directly to CDC_COUNTY_EXCLUSIONS?

**Review by mikelehen — APPROVED** [review]
Looks good, but with a typo and a question.

File: libs/datasets/combined_datasets.py (hunk 1 above)
Comment (smcclure17) [reply libs/datasets/combined_datasets.py:162]: oh no... 🤦 😭

File: libs/datasets/combined_datasets.py (hunk 2 above)
Comment (smcclure17) [reply libs/datasets/combined_datasets.py]: right, that would make much more sense

done in 49aa642bbf27ef04df1df3e75f4903193cea8f70

**Pull request merged**